packages/component/src/Styles/createStyleSet.js

```diff
@@ -19,6 +19,7 @@ import createRootStyle from './StyleSet/Root';
 import createScrollToEndButtonStyle from './StyleSet/ScrollToEndButton';
 import createSendBoxButtonStyle from './StyleSet/SendBoxButton';
 import createSendBoxStyle from './StyleSet/SendBox';
+import createSendBoxTextareaStyle from './StyleSet/SendBoxTextarea';
```

Comment:
Might want to change all of these names / paths to createSendBoxText**A**reaStyle

Reply:
I'm on the fence about this

Reply:
I’m on the fence, too. I originally had it as ‘TextArea’, but then I changed it to ‘Textarea.’